{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/conftest.py

```diff
@@ -1,12 +1,37 @@
 # pylint: disable=redefined-outer-name
 # flake8: noqa: E800
 import os
-from collections.abc import Generator
+from typing import Generator
```

Comment:
Aha! Didn't know this was in the `typing` module. Much cleaner! Even though they're functionally the same, that simple change illustrates what the import is actually being used for.